docs/basics.md

```diff
@@ -539,6 +544,47 @@ Feature('Complex JS Stuff').retry(3);
 Every Scenario inside this feature will be rerun 3 times.
 You can make an exception for a specific scenario by passing the `retries` option to a Scenario.
 
+### Rerunning Flaky Tests Multiple Times
```

Comment:
@DavertMik I would remove "Rerunning Flaky Tests Multiple Times" from "basics.md" and just leave it in "advanced.md", currently it seems to be duplicated to me

Reply:
Yeah, I put this twice but this is not basics information